lib/std/debug.zig

```diff
@@ -342,6 +342,15 @@ pub fn assert(ok: bool) void {
     if (!ok) unreachable; // assertion failure
 }
 
+/// This function is like `assert` but can only invoke undefined behaviour if
```

Comment:
the point of debug modes safety checks is that its *not* UB, it'll panic instead

Reply:
Thanks. Updated!